dcp_prototype/backend/wrangling/migrations/utils/util.py

```diff
@@ -12,3 +18,37 @@ def merge_dictionary_into(dict1: Dict, dict2: Dict):
         else:
             merged_dictionary[key] = value if type(value) is list else [value]
     return merged_dictionary
+
+
+def list_files_in_bucket(bucket: str, key: str, continuation_token: str = None, files: typing.Dict[str, str] = {}, checksums: bool=False) -> Dict[str, str]:
```

Comment:
Could you please write unit tests for this function? There is a respective unit test file in tests/

Reply:
discussed offline